jekyll/_cci2/creating-orbs.md

```diff
@@ -2,239 +2,141 @@
 
-```sh
-$ tree
-.
-├── config.yml
-└── foo
-    ├── bar
-    │   └── @baz.yml
-    ├── foo.yml
-    └── subtree
-        └── types.yml
+![Manually approve publishing development orbs]({{site.baseurl}}/assets/img/docs/orb-publish-approval.png)
 
-3 directories, 4 files
-```
+When we place a job on-hold for manual approval, the workflow will wait for our physical input, via an alert prompt, before continuing. Once we authorize the manual approval, subsequent jobs will be authorized by us and may access our Restricted Context. This is especially useful when we want to allow open-source pull requests to build against our orb but gives us a chance to ensure there is no malicious code.
 
-The unix `tree` command is great for printing out folder structures. In the example tree structure above, the `pack` command will  map the folder names and file names to **yaml keys**  and the file contents as the **values** to those keys. Let's `pack` up the example folder from above:
+After the workflow has been approved, the [orb-tools/publish-dev](https://github.com/CircleCI-Public/Orb-Project-Template/blob/0354adde8405564ee7fc77e21335090a080daebf/.circleci/config.yml#L62) job will publish a development version of your orb twice:
 
-{% raw %}
-```sh
-$ circleci config pack foo
-```
+| Published Development Tag | Description|
+| ----------| -----------|
+| `<namespace>/<orb>@dev:<branch>`     | A development tag linked to the branch name. Useful for testing in your configs. |
+| `<namespace>/<orb>@dev:${CIRCLE_SHA1:0:7}`     | A development tag specific to this SHA. Used in the following workflow. |
+{: class="table table-striped"}
 
-```yaml
-version: 2.1
-bar:
-  baz: qux
-foo: bar
-subtree:
-  types:
-    ginkgo:
-      seasonality: deciduous
-    oak:
-      seasonality: deciduous
-    pine:
-      seasonality: evergreen
-```
-{% endraw %}
+You can learn more about the testing jobs on the [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs) page.
 
-#### Other Config Packing Capabilities
-{:.no_toc}
+#### integration-test_deploy
 
-A file beginning with `@` will have its contents merged into its parent folder level. This can be useful at the top level of an orb, when one might want generic `orb.yml` to contain metadata, but not to map into an `orb` key-value pair.
+[integration-test_deploy](https://github.com/CircleCI-Public/Orb-Project-Template/blob/0354adde8405564ee7fc77e21335090a080daebf/.circleci/config.yml#L78) is the second and last workflow to fire in our orb development pipeline. This workflow is manually triggered via API at the end of the `test-pack` workflow. This new workflow will have access to the unique [development version]({{site.baseurl}}/2.0/orb-concepts/#orb-versions-development-vs-production-vs-inline) of our orb that was produced by the `test-pack` workflow.
```

Comment:
Is it manually triggered? How does this happen?

Reply:
I think "automatically, via our API" is better wording here. It's not a manual process, but the last job does have to reach out to the API to start the test workflow.